vaadin-radio-button-flow-parent/vaadin-radio-button-flow/src/main/java/com/vaadin/flow/component/radiobutton/RadioButton.java

```diff
@@ -61,4 +72,44 @@ private Label appendLabelElement() {
         getElement().appendChild(label.getElement());
         return label;
     }
+
+    /**
+     * True if the radio button is checked.
+     *
+     * @return the {@code checked} property from the webcomponent
+     */
+    @Synchronize(property = "checked", value = "checked-changed")
+    boolean isCheckedBoolean() {
+        return getElement().getProperty("checked", false);
+    }
+
+    /**
+     * True if the radio button is checked.
+     *
+     * @param checked
+     *            the boolean value to set
+     */
+    void setChecked(boolean checked) {
+        getElement().setProperty("checked", checked);
+    }
+
+    /**
+     * If true, the user cannot interact with this element.
+     *
+     * @return the {@code disabled} property from the webcomponent
+     */
+    protected boolean isDisabledBoolean() {
```

Comment:
Should this be made package-private, like the other API here? Seems that the method isn't used at all, so could even consider removing it.

Reply:
Removed.